### PR TITLE
Cleanup: s/depreciated/deprecated/g

### PR DESCRIPTION
**Pull request by thockin**
Super nit-picky, but I couldn't un-see it.

/kind cleanup
```release-note
NONE
```

### CODE REVIEW

**Review by pacoxu — APPROVED** [review]
/lgtm

File: vendor/sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1_21_2/swagger.pb [hunk 1]
```diff
@@ -36978,7 +36978,7 @@ containers
 enableServiceLinks�"�EnableServiceLinks indicates whether information about services should be injected into pod's environment variables, matching the syntax of Docker links. Optional: Defaults to true.�	
 boolean
 �
-serviceAccount�"sDeprecatedServiceAccount is a depreciated alias for ServiceAccountName. Deprecated: Use serviceAccountName instead.�
+serviceAccount�"sDeprecatedServiceAccount is a deprecated alias for ServiceAccountName. Deprecated: Use serviceAccountName instead.�
```
Comment (pacoxu) [inline vendor/sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1_21_2/swagger.pb:36981]: Leave this to kustomize owners to fix in its repo. We cannot update vendor directly 😄

**Comment by pacoxu** [comment]
/assign fsmunoz
for changelog approval.

File: vendor/sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1_21_2/swagger.pb (hunk 1 above)
Comment (charles-chenzz) [reply vendor/sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1_21_2/swagger.pb:36981]: seems like related to kustomize/kyaml doc?

Comment (thockin) [reply vendor/sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1_21_2/swagger.pb:36981]: Woops, I missed that - should not be in this PR

**Comment by thockin** [comment]
Pushed without touching vendor'ed files

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**